test_runner/src/main/kotlin/ftl/args/yml/Type.kt

```diff
@@ -0,0 +1,17 @@
+package ftl.args.yml
+
+import ftl.run.exception.FlankGeneralError
+
+enum class Type(val ymlName: String) {
+    INSTRUMENTATION("instrumentation"), ROBO("robo"), GAMELOOP("game-loop");
+
+    companion object {
+        fun fromString(stringVal: String): Type {
+            val filtered = values().filter { it.ymlName == stringVal }
+            if (filtered.isEmpty()) {
+                throw FlankGeneralError("Unsupported Type given `$stringVal` only [${values().joinToString(",")}] supported.")
```

Comment:
```suggestion
                throw FlankGeneralError("Unsupported Type given `$stringVal` only [${values().joinToString(", ") { it.ymlName }}] supported.")
```

I think it would be better to display yaml names
```
Unsupported Type given `instrumentationasda` only [instrumentation, robo, game-loop] supported.
```
instead of
```
Unsupported Type given `instrumentationasda` only [INSTRUMENTATION,ROBO,GAMELOOP] supported.
```

Reply:
nice spot, fixed 👍